### DISCUSSION TITLE
CyberGhost - How to select correct cipher after "ncp-disable" OpenVPN option was removed

### DISCUSSION THREAD

**Discussion by cluelesscactus** [qdm12/gluetun#627] [Q&A] [answered]
Hey everyone,

With the recent change that removed "ncp-disable" due to it being deprecated ([linked here](https://github.com/qdm12/gluetun/commit/0730b6db6ef0d10886467f5f8307bb71f9d7fbbb#diff-493b1af9d1249e552648d0065d6553346f926bc6881f1e0ce3b32ff88abce6d7L37)) I'm unable to successfully connect to any CyberGhost servers.  I get the below error during vpn startup

`2021/09/14 20:21:44 INFO openvpn: OPTIONS ERROR: failed to negotiate cipher with server.  Add the server's cipher ('AES-128-GCM') to --data-ciphers (currently 'AES-256-CBC') if you want to connect to this server.`

I noticed another thread on [Windscribe OpenVPN TLS Verification ](https://github.com/qdm12/gluetun/discussions/619#discussioncomment-1321412) that stated to add the value needed as an env (example here would be `OPENVPN_CIPHER=AES-128-GCM`) and this DOES WORK.  

I just wanted to double check if this is the "correct" way that I should be doing this and also leave this here incase anyone else runs into the issues with the new image that was released today (Sept 14th)

**Comment by qdm12** [comment, marked as answer]
I changed the default cipher to AES-128-GCM in 80f25c34e5bbc0aada856e6ce4f62e20acd08633 thanks for taking the time to report the issue! Yes `ncp-disable` was meant to be a retro-compatible option for negotiating ciphers, it's better and more secure to set the right cipher(s) on the client directly. Plus it will become an invalid option when OpenVPN 2.6 is supported in the future.

**Reply by cluelesscactus** [reply]
Thanks!  I pulled the latest image and removed the env var and it's working as expected now.  Thanks again!
